tests/integration_tests/data_steward/bq_utils_test.py

```diff
@@ -77,12 +77,8 @@ def setUp(self):
             }
         ]
         self.DT_FORMAT = '%Y-%m-%d %H:%M:%S'
-        self._empty_bucket()
-
-    def _empty_bucket(self):
-        bucket_items = gcs_utils.list_bucket(self.hpo_bucket)
-        for bucket_item in bucket_items:
-            gcs_utils.delete_object(self.hpo_bucket, bucket_item['name'])
+        self.client = StorageClient()
```

Comment:
If you're talking about what I think your talking about:

That `self.client = StorageClient()` mean while:

```
sc = StorageClient()
sc_bucket = sc.get_bucket(self.hpo_bucket)
```
later down.   Seems it happens 5ish times... Git blame says Luis in DC-1969

Reply:
That is what I  am referencing.  Didn't know if you had plans to clean this up in a later PR or if you might do it as part of this one.

Reply:
I can just get the change in there.  Pretty straightforward.

Reply:
I made the changes, confirm it with a diff.  - we can let the tests chug away since it'll only take ~25 minutes now.  Or if you feel comfortable, just review and accept the change.